integration/setup/teardown.go

```diff
@@ -1,156 +0,0 @@
-// +build k8srequired
-
-package setup
-
-import (
-	"context"
-	"fmt"
-	"time"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/cloudformation"
-	"github.com/giantswarm/backoff"
-	"github.com/giantswarm/microerror"
-
-	"github.com/giantswarm/aws-operator/integration/env"
-)
-
-func teardown(ctx context.Context, config Config) error {
-	var err error
-	var errors []error
-
-	{
-		releases := []string{
-			fmt.Sprintf("%s-aws-operator", config.Host.TargetNamespace()),
-			fmt.Sprintf("%s-cert-operator", config.Host.TargetNamespace()),
-			fmt.Sprintf("%s-node-operator", config.Host.TargetNamespace()),
-
-			fmt.Sprintf("e2esetup-awsconfig-%s", env.ClusterID()),
-			fmt.Sprintf("e2esetup-certs-%s", env.ClusterID()),
-		}
-
-		for _, release := range releases {
-			err = config.Release.EnsureDeleted(ctx, release)
-			if err != nil {
-				config.Logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("failed to delete release %#q", release), "stack", fmt.Sprintf("%#v", err))
-				errors = append(errors, microerror.Mask(err))
-			}
-		}
-	}
-
-	{
-		stackName := "host-peer-" + env.ClusterID()
-		wait := true
-
-		// TODO extract cloudformation stack client which can be also
-		// used in cloudformation resource.
-		// Issue: https://github.com/giantswarm/giantswarm/issues/3783.
-		err := ensureDeletedStack(ctx, config, stackName, wait)
-		if err != nil {
-			config.Logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("failed to delete host VPC stack"), "stack", fmt.Sprintf("%#v", err))
-			errors = append(errors, microerror.Mask(err))
-		}
-	}
-
-	{
-		// TODO there should be error handling for the framework teardown.
-		config.Host.Teardown()
-	}
-
-	if len(errors) > 0 {
-		return microerror.Mask(errors[0])
-	}
-
-	return nil
-}
-
-func ensureDeletedStack(ctx context.Context, config Config, stackName string, wait bool) error {
```

Comment:
Why remove this as well?

Reply:
It is not used. It is dead code. I think this was once intended to delete the host peer stack. This is done by the CI cleaner though. The mechanism here is totally broken as it seems.

Reply:
Okay, I am pretty sure that we still try to delete the stack after the e2e test. 
If we fully rely on the CI cleaner, then we limit the amount of E2E tests we can run per hour.
Each stack has one VPC peering, stacks only get deleted every hour by the ci-cleaner = 25 Peerings is the maximum AFAIK.

Maybe I am missing that we delete it somewhere else but couldn't find it quickly right now.

Reply:
I understand and agree with you. It is only that, it does not look like that the code I remove here is run at all right now. So what you describe is currently the case assumedly.

Reply:
Can you create a card on the releng board? I want to remember this and make sure to track it.

Reply:
Created a card.